Documentation/Architecture/ConfigurationAndProfiles.md

```diff
@@ -0,0 +1,85 @@
+# Configuration and profiles
+
+Not every single consumer of the MRTK will want it to behave the same way - some will want to have the spatial
+mesh running when on AR devices that support it. Some may want the diagnostic visualization on all the time,
+and some may only want it on when the user says a voice command. 
+
+The MRTK needs to be configurable in order to support a wide range of those requirements, and it uses a concept
+called 'profiles' to accomplish this.
+
+## What is a profile?
+
+A profile is a serialized asset (an instance of a ScriptableObject) which contains properties that configure
+the initialization and runtime behavior of an MRTK service. For example, the MRTK has a camera service, which
+will apply different properties to the main camera, depending on whether or not the display is transparent
+(like in the case of a HoloLens) or opaque (like in the case of a VR headset). The camera service is given
+a [camera profile](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit/Definitions/MixedRealityCameraProfile.cs),
+which contains those different transparent vs. opaque settings.
+
+Profiles can be more useful in more complicated cases, like in the [InputSystem](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputSystemProfile.cs).
+Some of the properties on that profile (such as the MixedRealityInputDataProviderConfiguration entities)
+control the objects that will be instantiated at runtime - this is how the input system knows how to create
+OpenVR, WMR and Unity input subsystems. This profile is not just a set of properties that configures if
+a particular input sub-feature is enabled or disabled - it's also an injection mechanism that the MRTK
+will use to "new" other classes at runtime (for example, the input system profile contains a list of
+'Input Data Providers' which has serialized type information - these objects are instantiated by the
+input system at runtime)
+
+Profiles also affect (and are no limited to) things like:
+
+1) What controller mappings are
+2) What types of pointers (these are discussed in the [Input System](InputSystem.md) overview)
+   are associated with various controllers.
+3) Which subset of the diagnostic and performance visualization tools are enabled.
+
+Profile configurations are initially greyed out because they're set up with MRTK's default profiles.
+They can only be modified after cloning to ensure that customized profiles won't be lost after a
+MRTK version update.
+
+## Where do I modify profiles?
+
+While profiles can be individually updated (by going to the serialized asset of the ScriptableObject),
```

Comment:
nit: can you use modified here as well? I didn't see that the first time i read through it.